src/main/java/com/solr/clientwrapper/infrastructure/solrbean/SolrCollectionIndex.java

```diff
@@ -0,0 +1,54 @@
+package com.solr.clientwrapper.infrastructure.solrbean;
+
+import java.util.List;
+
+import org.apache.solr.client.solrj.beans.Field;
+import org.springframework.data.solr.core.mapping.SolrDocument;
+import org.springframework.stereotype.Component;
+
+//@SolrDocument(collection = "techproducts")
+@Component
+public class SolrCollectionIndex {
+	
+	private String id;
+	private String name;
+	private List<String> features;
+	private List<String> cat;
```

Comment:
rename cat attribute

Reply:
Done. Renamed it as per it's definition.